packages/mobile/src/feature/login/login_screen.tsx

```diff
@@ -7,23 +8,37 @@ import { useDispatch, useSelector } from 'react-redux'
 import { AppButton } from '../../widgets/app_button/app_button'
 import { AppInput } from '../../widgets/app_input/app_input'
 import { userRequest } from 'presentation'
+import { useNavigation } from '@react-navigation/native'
+import RoutePaths from '../../navigation/router_path'
 
 const LoginScreen = () => {
-  const [username, setUsername] = useState<string>('')
-  const [password, setPassword] = useState<string>('')
+  const [username, setUsername] = useState<string>('systemadmin@aparajitha.com')
+  const navigation = useNavigation<any>()
+  const [password, setPassword] = useState<string>('admin@123')
   const [loadingState, setLodingState] = useState<boolean>(false)
   const dispatch = useDispatch()
-  const data = useSelector<any>(state => state.loginData)
+  const loginData: any = useSelector<any>(state => state.loginData)
   const saveData = () => {
-    setLodingState(true)
     const data = {
       email: username,
       password: password
     }
-    console.log('ha')
-    // redux dispatch will call  here
-    dispatch(userRequest(data))
+    if (username == '' || password == '') {
+      alert(i18n.t('noInput'))
+    } else {
+      dispatch(userRequest({ data: data }))
+    }
   }
+
+  useEffect(() => {
+    if (loginData?.status == 1) setLodingState(true)
```

Comment:
can we create a enum/constant to define the statuses?

Reply:
created enum for same in utils folder inside packages/mobile/utils/status.ts